Subject: [PATCH] Add support for MSVC ABI from `@llvm`
---
Index: rs/platforms/triples.bzl
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/rs/platforms/triples.bzl b/rs/platforms/triples.bzl
--- a/rs/platforms/triples.bzl	(revision afe82a1ab8ec161a59aa23b495780f56b8905afe)
+++ b/rs/platforms/triples.bzl	(revision f5545ff7c7abdc8f67b8cf3b5a29c41e1c8fb604)
@@ -10,6 +10,12 @@
     "msvc": "@rules_rs//rs/platforms/constraints:windows_msvc",
 }

+_WINDOWS_LLVM_ABI_CONSTRAINTS = {
+    "gnu": "@llvm//constraints/abi:gnu",
+    "gnullvm": "@llvm//constraints/abi:gnullvm",
+    "msvc": "@llvm//constraints/abi:msvc",
+}
+
 def triple_to_constraint_set(target_triple):
     constraints = _triple_to_constraint_set(target_triple)
     t = triple(target_triple)
@@ -23,6 +29,7 @@
             constraints.append("@llvm//constraints/libc:gnu.2.28")
     elif t.system == "windows":
         constraints.append(_WINDOWS_ABI_CONSTRAINTS[t.abi])
+        constraints.append(_WINDOWS_LLVM_ABI_CONSTRAINTS[t.abi])

         # Rust links MSVCRT for both GNU Windows target specs:
         # https://github.com/rust-lang/rust/blob/c935696dd07ca51e6fba2f6579919eea2a50863b/compiler/rustc_target/src/spec/base/windows_gnullvm.rs#L19
